### PR TITLE
fix(bookmarks-panel): hide Upload button

### PR DESCRIPTION
**Pull request by mspivak-actionengine**

### CODE REVIEW

File: src/components/bookmarks-panel/bookmarks-panel.spec.tsx [hunk 1]
```diff
@@ -177,7 +177,8 @@ describe("BookmarksPanel", () => {
     userEvent.click(uploadOption);
     const body = document.getElementsByTagName("body")[0];
     const popover = body.childNodes[1];
-    const uploadButton = getByText("Upload");
+//    const uploadButton = getByText("Upload");
```
Comment (belom88) [inline src/components/bookmarks-panel/bookmarks-panel.spec.tsx:180]: remove

File: src/components/bookmarks-panel/bookmarks-panel.spec.tsx [hunk 2]
```diff
@@ -207,7 +208,8 @@ describe("BookmarksPanel", () => {
...
```
Comment (belom88) [inline src/components/bookmarks-panel/bookmarks-panel.spec.tsx:211]: ditto

File: src/components/bookmarks-panel/upload-panel-item.tsx [hunk 3]
```diff
@@ -28,21 +28,26 @@ const Title = styled.div`
...
-  justify-content: space-between;
+  flex-direction: row;
+  justify-content: ${(props) => props.justify};
 `;
 
 type UploadPanelItemProps = {
   title?: string;
   children: React.ReactNode;
+  okButtonText?: string;
+  cancelButtonText?: string;
```
Comment (belom88) [inline src/components/bookmarks-panel/upload-panel-item.tsx:41]: I don't see any reason to have these props. ok is always "Next", cancel is always "Cancel". The component is not generic so there is no sense to add these properties.

File: src/components/bookmarks-panel/unsaved-bookmark-warning.tsx [hunk 4]
```diff
@@ -32,7 +32,7 @@ export const UnsavedBookmarkWarning = ({
   onConfirmWarning,
 }: ExistedLayerWarningProps) => {
   return (
-    <UploadPanelItem onCancel={onCancel} onConfirm={onConfirmWarning}>
+    <UploadPanelItem onCancel={onCancel} onConfirm={onConfirmWarning} cancelButtonText={"Cancel"}  okButtonText={"Next"}>
```
Comment (belom88) [inline src/components/bookmarks-panel/unsaved-bookmark-warning.tsx:35]: Long line can be formatted

File: src/components/bookmarks-panel/bookmarks-panel.spec.tsx (hunk 1 above)
Comment (mspivak-actionengine) [reply src/components/bookmarks-panel/bookmarks-panel.spec.tsx]: Done.

File: src/components/bookmarks-panel/bookmarks-panel.spec.tsx (hunk 2 above)
Comment (mspivak-actionengine) [reply src/components/bookmarks-panel/bookmarks-panel.spec.tsx]: Done.

File: src/components/bookmarks-panel/upload-panel-item.tsx (hunk 3 above)
Comment (mspivak-actionengine) [reply src/components/bookmarks-panel/upload-panel-item.tsx]: Done.

File: src/components/bookmarks-panel/unsaved-bookmark-warning.tsx (hunk 4 above)
Comment (mspivak-actionengine) [reply src/components/bookmarks-panel/unsaved-bookmark-warning.tsx]: Unnecessary args are removed.

**Review by belom88 — APPROVED** [review]

**Pull request merged**